addons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,40 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+
+# TODO enable CORS!
+
+oc login -u system:admin
```

Comment:
@LalatenduMohanty hmm, `oc --as system:admin` doesn't seem to work
```
$ oc --as system:admin  get oauthclient
error: error processing template "fabric8/fabric8-oauthclient-template": User "developer" cannot "impersonate" "systemusers.user.openshift.io" with name "system:admin" in project ""

or when trying to use it instead of `oc login -u system:admin` I now get:
```
oc --as system:admin adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:fabric8:init-tenant
Error from server (Forbidden): User "developer" cannot "impersonate" "systemusers.user.openshift.io" with name "system:admin" in project ""
```